portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,84 @@
+package com.google.sps.data;
+
+import javax.servlet.http.HttpServletRequest;
+import com.google.appengine.api.datastore.Entity;
+
+/**
+ * A class for storing a Comment.
+ */
+public class Comment {
+    private String author;
+    private String text;
+    private long timestamp;
+
+    /**
+     * No argument constructor.
+     * Set fields with setter methods.
+     */
+    public Comment() {
+        this.timestamp = System.currentTimeMillis();
+    }
+
+    public Comment(String author, String text) {
+        this();
```

Comment:
Doesn't seem like the original comment constructor is being used without immediate setter calls. If that's the case, it might be simpler to remove this constructor, and add set this.timestamp in the constructor below.

Reply:
Fixed. Removed no arg constructor and now using other constructor for everything.